platform-api/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/type/AbstractAttribute.java

```diff
@@ -33,7 +33,7 @@ protected Attribute(String projectType, String name, String description, boolean
 
 
     public String getId() {
-        return projectType+":"+name;
+        return projectType + ":" + name;
```

Comment:
Just a hint: use character literals for concatenations like this as it uses `StringBuilder` in background and character append worth less than string append.